dlang.org.ddoc

```diff
@@ -301,9 +301,9 @@ WIKI=$(TITLE)
 _=
 
 XREF=$(XREF2 $1, $2, $(D std.$1.$2))
-XREF2=$(SPANC libref, $(AHTTP dlang.org/phobos/std_$1.html#$2, $(TAIL $+)))
+XREF2=$(SPANC libref, $(A /phobos/std_$1.html#$2, $(TAIL $+)))
 XREF_PACK=$(XREF_PACK_NAMED $1, $2, $3, $(D std.$1.$2.$3))
-XREF_PACK_NAMED=$(SPANC libref, $(AHTTP dlang.org/phobos/std_$1_$2.html#$3, $4))
+XREF_PACK_NAMED=$(SPANC libref, $(A /phobos/std_$1_$2.html#$3, $4))
```

Comment:
Absolute paths (starting with `/`) are bad. They don't work when you open `.html` files directly, without a web server in between, or when the site is built in a sub-directory. I'd expect the links to be broken in the documentation tester output, but it spits 500s right now, so can't check.

Instead of a leading `/`, use the `ROOT_DIR` macro to get a relative path to the site root: `$(ROOT_DIR)phobos/...`